yarn-project/circuit-types/src/logs/l1_payload/encrypted_log_incoming_body/encrypted_event_log_incoming_body.test.ts

```diff
@@ -31,32 +30,4 @@ describe('encrypt log incoming body', () => {
 
     expect(recreated.toBuffer()).toEqual(body.toBuffer());
   });
-
-  it('encrypt an event log incoming body, generate input for noir test', () => {
```

Comment:
This test was nuked because it no longer has a Noir alternative since it got merged with the note one --> this is fine as we now only accept plaintext on input.